addons/resource.language.en_gb/resources/strings.po

```diff
@@ -21110,7 +21113,28 @@ msgctxt "#36637"
 msgid "Note that add-ons installed from zip (excluding served repositories) will not auto-update and must be manually updated. Would you like to proceed?"
 msgstr ""
 
-#empty strings from id 36638 to 36898
+#. Description of setting with label #24163 "Remove orphaned dependencies recursively"
+#: system/settings/settings.xml
+msgctxt "#36638"
+msgid "Remove all modules and support libraries that have been automatically installed as a dependency to other add-ons and are in an orphaned state."
+msgstr ""
+
+#: xbmc/addons/AddonSystemSettings.cpp
+msgctxt "#36639"
+msgid "Orphaned dependencies"
+msgstr ""
+
+#: xbmc/addons/AddonSystemSettings.cpp
+msgctxt "#36640"
+msgid "The following orphaned dependency add-ons were removed from your system: {0:s}."
+msgstr ""
+
+#: xbmc/addons/AddonSystemSettings.cpp
+msgctxt "#36641"
+msgid "No orphaned dependency add-ons found."
```

Comment:
This reads strange. Do you mean "No add-ons with orphaned dependencies found." ?

Reply:
no it should say: there are no more orphaned add-ons left that i could delete

Reply:
So: `No orphaned dependency add-ons to delete`?

Reply:
done